src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+                System.out.println("Swee! One less task to go! Removing...\n" + removed);
+            } else { //Task Creation
+                Task task = null;
+                if (isToDo(input)) {
+                    task = new ToDo(input);
+                    list.add(task);
+                } else if (isDeadline(input)) {
+                    int index = input.indexOf(" /by ");
+                    task = new Deadline(input.substring(0, index), input.substring(index + 5));
+                    list.add(task);
+                } else if (isEvent(input)) {
+                    int fromdex = input.indexOf(" /from ");
+                    int todex = input.indexOf(" /to ");
+                    task = new Event(input.substring(0, fromdex), input.substring(fromdex + 7, todex), input.substring(todex + 5));
+                    list.add(task);
+                } else {
+                    throw new DukeException("What are you saying?\n" + "Please input a task with either todo, deadline or event prefixed!");
+                }
+                System.out.println("Roger. This task has been added:\n" + "  " + task.toString());
+                System.out.println("Now you have " + list.size() + " tasks in your list.");
+            }
+        }
+    }
+
+    public static boolean isMark(String input, int listSize) {
+        if (input.length() >=  6 && input.startsWith("mark ") && isNumeric(input.substring(5))) {
+            int taskindex = Integer.parseInt(input.substring(5)) - 1;
+            return !(taskindex < 0 || taskindex > listSize - 1);
+        }
+        return false;
+    }
+
+    public static boolean isUnMark(String input, int listSize) {
+        if (input.length() >=  8 && input.startsWith("unmark ") && isNumeric(input.substring(7))) {
+            int taskindex = Integer.parseInt(input.substring(7)) - 1;
+            return !(taskindex < 0 || taskindex > listSize - 1);
+        }
+        return false;
+    }
+
+    public static boolean isDelete(String input, int listSize) {
+        if (input.length() >=  8 && input.startsWith("delete ") && isNumeric(input.substring(7))) {
+            int taskindex = Integer.parseInt(input.substring(7)) - 1;
+            return !(taskindex < 0 || taskindex > listSize - 1);
+        }
+        return false;
+    }
+
+    public static boolean isToDo(String input) throws DukeException {
+        if (input.length() >= 4 && input.startsWith("todo")) {
+            if (input.equals("todo") || input.substring(4).isBlank()) {
+                throw new DukeException("TODO needs a description!");
+            } else return input.startsWith("todo ");
+        }
+        return false;
+    }
+
+    public static boolean isDeadline(String input) throws DukeException {
+        if (input.length() >= 8 && input.startsWith("deadline")) {
+            if (input.equals("deadline") || input.substring(8).isBlank() || input.equals("deadline /by") || input.equals("deadline /by ")) {
```

Comment:
I would recommend breaking this statement into 2 lines to make the code look more neat.